gitrevise/utils.py

```diff
@@ -121,7 +121,9 @@ def run_editor(
         if comments:  # If comments were provided, write them after the text.
             handle.write(b"\n")
             for comment in textwrap.dedent(comments).splitlines():
-                handle.write(commentchar + b" " + comment.encode("utf-8") + b"\n")
+                handle.write(commentchar)
+                handle.write((b" " + comment.encode("utf-8")) if comment else b"")
```

Comment:
Looks good, but I think it'd be more clear if we skip the entire `write` call if there's no comment line to write out :-)

```suggestion
                if comment:
                    handle.write(b" " + comment.encode("utf-8"))
```

Reply:
Ok! Tried to get around yet one more indentation level :)